app/controllers/service_uploads_controller.rb

```diff
@@ -45,15 +45,9 @@ def create
 
```

Comment:
@alexsoble I want to make sure service batch uploads continues to work after these changes.